en/Live preview update.md

```diff
@@ -1,61 +1,59 @@
 
-### The options
+This is what your editing experience used to look like:
 
-We understand that this change is not for everyone. You have a few options below (you can change your mind anytime!):
+![[Source view.png]]
 
-#### 1. Hide formatting
+And here's the same note using Live Preview:
 
-**If you**: like to hide formatting and see the images and embeds right in the editor, we suggest you try the new Live Preview.
+![[Live Preview.gif]]
+Live Preview gives you an improved editing experience that lets you see how your Markdown note will look like, without having to switch between Edit and Preview.
 
-**What to do**:
+To enable Live Preview for an existing vault:
 
 1. Open **Settings**.
-2. In the sidebar, click **Editor** if it's not already open.
-3. Under **General**, turn off **Use legacy editor** if it's not off already. After turning it off, click on the **Relaunch** button for it to take effect.
+2. In the sidebar, click **Editor**.
+3. Under **General**, disable **Legacy Editor**. 
+4. Click on the **Relaunch** button for the change to take effect.
 4. After Obsidian restarts, open **Settings**.
 5. In the sidebar, click **Editor**.
 6. Under **General**, set **Default editing mode** to **Live Preview**.
-7. All newly opened panes will use Live Preview at this point. If you already have some panes open, they will show **Source View**. To change this, in the **More options** menu of the pane, select **Toggle Live Preview/Source View**. Alternatively, you can open [[Command palette]] and search for the **Toggle Live Preview/Source View** command.
 
-#### 2. I like seeing Markdown
+When you open a new pane, it'll use Live Preview by default. If you already have some panes open, they'll be in _Source mode_.
 
-**If you**: like to continue seeing Markdown and would also like an updated editor engine, you can use the the new Source View.
+## Edit notes using Live Preview or Source mode
 
-**What to do**:
+We think you're going to love Live Preview. But if you want to keep your current editing experience, we've got you covered. You can keep editing your notes the same way as before by switching to the new Source mode.
 
-1. Open **Settings**.
-2. In the sidebar, click **Editor** if it's not already open.
-3. Under **General**, turn off **Use legacy editor** if it's not off already. After turning it off, click on the **Relaunch** button for it to take effect.
-
-**Pros**: There are some new features like spellchecker and text substitution on macOS. Vim mode is also supported.
+To toggle between Live Preview and Source mode:
 
-**Cons**: Some community plugins might not be fully compatible yet; Vim mode is experimental.
+1. In the upper right corner of the pane, click **More options** (the three dots).
+2. Select **Toggle source/live preview mode**.
```

Comment:
this toggles live preview/source view for a single pane, might want to include the instructions to switch the default as well, so they don't need to keep switching?

default setting is in settings - editor - general

Reply:
It seems to me like this is covered in the first instructions?: https://github.com/obsidianmd/obsidian-docs/pull/210/files#diff-4f0136a2748fefa17331da58c2bdb348a41769cfe6fc6097231162865c5d09fdR29

Reply:
Oh, you mean instructions for setting the default to source mode (instead of Live Preview)? Maybe since we already covered default editing mode, we can add something like:

> You can also toggle the editing view by opening the Command palette and searching for the Toggle source/live preview mode command, or change it permanently by setting **Default editing mode** to **Source mode**.

Reply:
Yes, that would make it more clear!